client/errors.go

```diff
@@ -72,8 +101,7 @@ func (e volumeNotFoundError) Error() string {
 // IsErrVolumeNotFound returns true if the error is caused
 // when a volume is not found in the docker host.
 func IsErrVolumeNotFound(err error) bool {
-	_, ok := err.(volumeNotFoundError)
-	return ok
+	return IsErrNotFound(err)
```

Comment:
The nice thing about this approach is that we don't necessarily need to add it in this PR. We just define a new interface method that exports the object type.

Is there a direct need for this as part of this PR?